### PR TITLE
Partners page milestone1

### PR DESCRIPTION
**Pull request by hoangviet1993**
After Jordan handed me the Sketch folder, I had a much clearer vision on how the page should be structured on mobile view. A lot less assuming! Still a lot of rough edges to buff out tho :)

I am current using one placeholder photo because the provided photo keeps breaking resolutions/ sizes. Will get back to it later once I actually understand what is going on!

### CODE REVIEW

File: pages/partnerships/partnerships.html [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+  </md-card>
+  <md-card layout-fill class="oppia-foundation-gray-background oppia-foundation-vertical-padding partnerships-page-conditional-horizontal-padding">
+    <div layout-gt-md="row" layout-xs="column" layout-align="space-between none" flex layout-fill>
+      <div hide-gt-md flex-gt-md="50" flex="100" layout="column" layout-align="center" layout-wrap>
+        <img src="https://dummyimage.com/512x512/000/fff" class="partnerships-page-place-holder-image">
+      </div>
+      <div flex-gt-md="50" flex="100">
+        <md-card-content layout="column">
+          <md-content layout="column" layout-align="center none" class="oppia-foundation-gray-background oppia-foundation-black-text">
+             <h3 class="md-display-1 partnerships-description-headline-padding">
```
Comment (seanlip) [inline pages/partnerships/partnerships.html]: Indent by 2 not 3

File: pages/partnerships/partnerships.html [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+            </p>
+            <p>
+              It’s a problem that isn’t limited to developing countries. In the united states, low income and rural students consistently show major achievement gaps compared to their more affluent peers.
+            </p>
+            <p>
+              The Oppia Foundation believes a digital classroom that provides access to interactive, engaging lessons is the key to closing these gaps in educational outcomes.
+            </p>
+          </md-content>
+      </md-card-content>
+      </div>
```
Comment (seanlip) [inline pages/partnerships/partnerships.html]: Something's wrong with the indentation (here and below)

File: pages/partnerships/partnerships.html [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+      <div hide-xs hide-sm hide-md flex-gt-md="50" flex="100" layout="column" layout-align="center" layout-wrap>
+        <img src="https://dummyimage.com/512x512/000/fff" class="partnerships-page-place-holder-image">
+      </div>
+  </div>
+  </md-card>
```
Comment (seanlip) [inline pages/partnerships/partnerships.html]: Ditto -- indentation problems. The closing tag should match the level of the opening tag; there should be two-space indentation.

File: pages/partnerships/partnerships.html [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+  <md-card layout-fill class="oppia-foundation-green-background partnerships-page-conditional-horizontal-padding oppia-foundation-vertical-padding">
+    <md-card-content layout-gt-md="row" layout-xs="column">
+      <img src="/static/images/partnerships_our_approach.png" class="partnerships-page-approach-image">
+      <md-content layout="column" layout-align="center none" class="oppia-foundation-green-background oppia-foundation-white-text partnerships-page-description-text-box">
+        <h3 class="md-display-1 partnerships-description-headline-padding">
+            Our Approach
```
Comment (seanlip) [inline pages/partnerships/partnerships.html]: Indent by 2 not 4.

File: pages/partnerships/partnerships.html [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+          </div>
+        </md-input-container>
+        <md-button class="oppia-foundation-button-horizontal-padding oppia-foundation-green-button" type="submit">Submit</md-button>
+      </form>
+    </md-card>
+  </md-card>
+</section>
+
+<style>
+  blockquote {
```
Comment (seanlip) [inline pages/partnerships/partnerships.html]: If this is generic it should go in the main CSS stylesheet.

Comment (hoangviet1993) [reply pages/partnerships/partnerships.html]: Upon closer inspection, I don't think I want to override blockquote in general so I turned this into a specific css class.

File: pages/partnerships/partnerships.html (hunk 1 above)
Comment (hoangviet1993) [reply pages/partnerships/partnerships.html]: Fixed!

File: pages/partnerships/partnerships.html (hunk 2 above)
Comment (hoangviet1993) [reply pages/partnerships/partnerships.html]: Fixed! I have checked each line below this line.

File: pages/partnerships/partnerships.html (hunk 3 above)
Comment (hoangviet1993) [reply pages/partnerships/partnerships.html]: Fixed. A lot of these came down to me copy-&-pasting and not getting to fix all of the incorrect indentations. 
Will try to catch these earlier/ more often.

**Comment by hoangviet1993** [comment]
Should match the mock better now!
Reg view
partnerships_page_reg_view
Mobile view
partnerships_page_mobile_view

**Comment by seanlip** [comment]
Thanks, @hoangviet1993! The desktop screenshots look good. But the mobile mocks look a bit cut off?

**Comment by hoangviet1993** [comment]
Ah looks like the Chrome plugin I am using to take whole page screenshot is having problem again. I have switched to a different one now. How does this look now?

Mobile:
invest_mobile_view

File: static/css/stylesheet.css [hunk 6]
```diff
@@ -326,6 +326,11 @@ Full-Width Styles
   zoom: 1;
 }
 
+.oppia-foundation-no-horizontal-padding {
+  padding-left: 0px;
```
Comment (seanlip) [inline static/css/stylesheet.css]: Prefer 0 to 0px (remove units when it's 0).

Comment (hoangviet1993) [reply static/css/stylesheet.css]: Thanks for spotting this :)

**Comment by hoangviet1993** [comment]
Merging this PR. Thanks again!

**Pull request merged**